trompace/mutations/audioobject.py

```diff
@@ -140,3 +140,60 @@ def mutation_delete_audio_object(identifier: str):
     """
 
     return format_mutation("DeleteAudioObject", {"identifier": identifier})
+
+
+def mutation_remove_audio_object_work_example(audio_object_identifier: str, work_identifier: str):
+    """Returns a mutation for creating removing a audio object as an example of a work.
+
+    Arguments:
+        audio_object_identifier: The unique identifier of the audio object.
+        work_identifier: The unique identifier of the work that the audio object is an example of.
+
+    Returns:
+        The string for the mutation for removing a audio object as an example of the work.
+    """
+
+    return format_link_mutation("RemoveAudioObjectExampleOfWork", audio_object_identifier, work_identifier)
+
+
+def mutation_merge_audio_object_encoding(audio_object_identifier_1: str, audio_object_identifier_2: str):
+    """Returns a mutation for creating merging a audio object as an encoding of another audio object.
+
+    Arguments:
+        audio_object_identifier_1: The unique identifier of the audio object that is encoding the other.
+        audio_object_identifier_2: The unique identifier of the audio object being encoded.
+
+    Returns:
+        The string for the mutation for merging a audio object as an encoding of another audio object.
+    """
+
+    return format_link_mutation("MergeAudioObjectEncoding", audio_object_identifier_1, audio_object_identifier_2)
+
+
+def mutation_remove_audio_object_encoding(audio_object_identifier_1: str, audio_object_identifier_2: str):
+    """Returns a mutation for creating removing a audio object as an encoding of another audio object.
+
+    Arguments:
+        audio_object_identifier_1: The unique identifier of the audio object that is encoding the other.
+        audio_object_identifier_2: The unique identifier of the audio object being encoded.
+
+    Returns:
+        The string for the mutation for removing a audio object as an encoding of another audio object.
+    """
+
+    return format_link_mutation("RemoveAudioObjectEncoding", audio_object_identifier_1, audio_object_identifier_2)
+
+
+# link AudioObject to Work (step 5)
+def mutation_merge_audio_object_work_example(audio_object_identifier: str, work_identifier: str):
```

Comment:
like the previous comment - check this with the mediaobject code. This is incorrect at the moment because the method is `work_example` and the mutation that it uses is `ExampleOfWork`. These are the inverse of each other, and so we should improve this.

Reply:
Also, be very careful about the order of the arguments (because we don't want to make the relation the wrong way around). In the mediaobject version we put the mediaboject as the first argument and the work as the second. Ensure that the CDR importer does the same.
https://github.com/trompamusic/trompa-ce-client/blob/0169d641a84e7dc4ab95eb29e8bc77c8e8d7aad6/demo/mediaobject.py#L89-L94